package.json

```diff
@@ -102,6 +102,7 @@
     "@angularclass/conventions-loader": "1.0.2",
     "@angularclass/hmr": "1.2.0",
     "@angularclass/hmr-loader": "3.0.2",
+    "@types/lodash": "^4.14.50",
```

Comment:
I'm using lodash, for deep cloning, in the SpaceService class, not just in the unit test.